Controller/Transaction/CommitWebpayM22.php

```diff
@@ -75,7 +75,7 @@ public function execute()
                 return $this->orderCanceledByUser($_POST['TBK_TOKEN'], $_POST['TBK_ID_SESION'], $orderStatusCanceled);
             }
             if (isset($_GET['TBK_TOKEN'])) {
-                $this->interactsWithFullLog->logWebpayPlusRetornandoDesdeTbkFujo2Error($_POST['TBK_ID_SESION']); // Logs
+                $this->interactsWithFullLog->logWebpayPlusRetornandoDesdeTbkFujo2Error($_GET['TBK_ID_SESION']); // Logs
```

Comment:
Sanitize this parameter before writing it to the log.

Reply:
to be included in another PR